sydent/db/invite_tokens.py

```diff
@@ -30,6 +30,19 @@ def storeToken(self, medium, address, roomId, sender, token):
         self.sydent.db.commit()
 
     def getTokens(self, medium, address):
+        """
+        Retrieves the pending invites tokens for this 3PID that haven't been delivered
+        yet.
+
+        :param medium: The medium of the 3PID to get tokens for.
+        :type medium: unicode
+        :param address: The address of the 3PID to get tokens for.
+        :type address: unicode
+
+        :return: A list of dicts, each containing a pending token and its metadata for
+            this 3PID.
+        :rtype: list[dict[str, unicode or dict[str, unicode]]
```

Comment:
FTR these probably want to be `Dict` and `List` (so that we can import them from `typing` and eventually turn on mypy)

Reply:
The reason I did that was that when not using `typing` PyCharm wouldn't know what `Dict` is but would know about `dict`.

Reply:
Though I agree that we should change that when we'll want to turn on mypy.

Reply:
Ah, fair enough